tools/_setup_generation/step_viselements.py

```diff
@@ -361,7 +360,18 @@ def build_doc(property, desc, indent: int):
                     dynamic = " (<i>dynamic</i>)"
                 doc = ""
                 if "doc" in desc:
-                    doc = str(desc["doc"]).replace("\n", f'\n{(indent+4)*" "}').replace("<br/>", f'<br/>\n{(indent+4)*" "}')
+                    doc = str(desc["doc"])
+                    # Update internal links
+                    new_doc = ""
+                    last_loc = 0
+                    INNER_HREF = re.compile(r"(?<=<a\shref=\")(#|\.)")
```

Comment:
\s+

Reply:
Unfortunately impossible in a positive lookbehind...